Modules/ModParser-3_0.lua

```diff
@@ -1582,6 +1582,27 @@ local specialModList = {
 	["socketed gems reserve no mana"] = { mod("ManaReserved", "MORE", -100, { type = "SocketedIn", slotName = "{SlotName}" }) },
 	["socketed skill gems get a (%d+)%% mana multiplier"] = function(num) return { mod("ExtraSkillMod", "LIST", { mod = mod("SupportManaMultiplier", "MORE", num - 100) }, { type = "SocketedIn", slotName = "{SlotName}" }) } end,
 	["socketed gems have blood magic"] = { flag("SkillBloodMagic", { type = "SocketedIn", slotName = "{SlotName}" }) },
+	["socketed gems have elemental equilibrium"] = function() 
+		--Currently hardcoded to what Elemental Equilibrium node is
+		local plus = 25
```

Comment:
I think it is fine to hardcode these values without refactoring into variables, like with the other mods.